Marlin/Configuration_adv.h

```diff
@@ -4342,7 +4342,7 @@
  * Extras for an ESP32-based motherboard with WIFISUPPORT
  * These options don't apply to add-on WiFi modules based on ESP32 WiFi101.
  */
-#if ENABLED(WIFISUPPORT)
+#if ANY(WIFISUPPORT, ESP3D_WIFISUPPORT)
```

Comment:
I am not familiar with Marlin's WiFi supoprt. Is the ESP32 WiFi101 mentioned in the comment related to WIFISUPPORT or ESP3D_WIFISUPPORT at all, or are those yet a third way to use WiFi?

Without knowing any better, a novice like me may assume that ESP32 WiFi101 is related to ESP3D_WIFISUPPORT. That is likely a bad assumption, meaning we could stand to improve the comments to make that harder to misinterpret.

Consider this optional and I welcome your thoughts on this.

Reply:
WiFi101 is a discontinued Arduino shield that apparently needed some kind of special code changes to Marlin other than needing just a serial port enabled like most WiFi addons.

It's a constant source of confusion for users, but my aim with this PR is to simply get ESP3D support on ESP-based motherboards working again.